.gitattributes

```diff
@@ -1,4 +1,5 @@
 * text eol=crlf
 *.md eol=lf
 *.yml eol=lf
-*.json eol=lf
\ No newline at end of file
+*.json eol=lf
+*.cs eol=crlf
```

Comment:
müssen Zeilen in *.cs  Dateien mit crlf enden?

Reply:
falscher alarm. der ziel-branch war falsch (develop statt main(richtig))